tests/integration/api/test_sc_test_jobs.py

```diff
@@ -0,0 +1,34 @@
+from tests.base import BaseTest
+
+from tenable_io.api.models import ScTestJob
+
+
+class TestScTestJobsApi(BaseTest):
+
+    def test_get(self, client, image):
+        containers = client.sc_containers_api.list()
+        assert len(containers) > 0, u'At least one image exists.'
+        container = [c for c in containers if image['name'].endswith(c.name) and
+                     c.digest == u'sha256:%s' % image['digest']][0]
+        assert container.name == image['name'], \
+            u'Image returned from the container list should match the image that was uploaded.'
+
+        jobs = client.sc_test_jobs_api.list()
+        assert len(jobs) > 0, u'At least one job exists.'
+        job = [j for j in jobs if j.container_id == container.id][0]
+        test_job = client.sc_test_jobs_api.get(job.job_id)
+        assert isinstance(test_job, ScTestJob), u'The method returns type.'
+        assert job.job_id == test_job.job_id, u'The job ID should match.'
+
+    def test_by_image(self, client, image):
+        job = client.sc_test_jobs_api.by_image(image['id'])
+        assert isinstance(job, ScTestJob), u'The method returns type.'
+
+    def test_by_image_digest(self, client, image):
+        job = client.sc_test_jobs_api.by_image(image['digest'])
+        assert isinstance(job, ScTestJob), u'The method returns type.'
+
+    def test_list(self, client):
```

Comment:
You should still inject `image` here to ensure that there is at least one job.

It might also be a good idea to combine `status/list` together in one test as `test_list_status` since the test above uses list api anyway. However, this is at your discretion.